src/cognito/index.ts

```diff
@@ -42,8 +42,8 @@ export async function withIdentityPoolId(
   return {
     getMapAuthenticationOptions: () => ({
       transformRequest: (url: string) => {
-        // Only sign aws URLs
-        if (url.includes("amazonaws.com")) {
+        // Only sign aws location service URLs
+        if (url.match("(http|https)://(.*).geo.(.*).amazonaws.com")) {
```

Comment:
Does this also apply to GovCloud? If not, we should fix that too.

Reply:
Should apply just to Maps. HTTPS-only.

Reply:
Good call out, will fix to handle the GovCloud geo-fips, and will tighten up to only maps + https.